packages/wallets/src/features/cfd/components/CompareAccountsCarousel/CompareAccountsCarousel.scss

```diff
@@ -0,0 +1,73 @@
+.wallets-compare-accounts-carousel {
+    position: relative;
+    --slide-spacing: 1rem;
+    --slide-size: 50%;
+    --slide-height: 19rem;
+    overflow: hidden;
+    &__viewport {
+        overflow: hidden;
+        width: 100%;
+        height: 100%;
+        @include mobile {
+            padding-bottom: 6rem;
+        }
+    }
+    &__container {
+        backface-visibility: hidden;
+        display: flex;
+        touch-action: pan-y;
+        flex-direction: row;
+        max-height: auto;
+        margin-left: calc(var(--slide-spacing) * -1);
+        transition: transform 0s ease-in-out;
+    }
+    &__slide {
+        flex: 0 0 var(--slide-size);
+        min-width: 0;
+        padding-left: var(--slide-spacing);
+        position: relative;
+        &__img {
```

Comment:
hmm, is this block or element? generally bem should have block__element--modifier, while this will result in block__element1__element2--modifier

basically we shouldn't have multiple levels of nesting when using BEM, so would be better to have:
```
&__slide { ... slide styles }
&__slide-image { ... slide image styles } 

also as a general rule of thumb, its good to keep structures flat,